components/cam/src/physics/cam/micro_p3_interface.F90

```diff
@@ -1001,6 +1001,12 @@ subroutine micro_p3_tend(state, ptend, dtime, pbuf)
     !==============
     ! TODO: get proper value for 'it' from time module
     dummy_out(:,:) = 0.0_rtype
+    prt_liq = 0.0_rtype
+    prt_sol = 0.0_rtype
+    prec_pcw = 0.0_rtype
+    snow_pcw = 0.0_rtype
+    vap_liq_exchange = 0.0_rtype
```

Comment:
vap_liq_exchange was removed by #421 so I think this last line needs to be deleted/you should merge in master before merging to master...

Reply:
I think `vap_liq_exchange` is there. You may be thinking of `vap_cld_exchange`, which indeed was removed recently.

Reply:
You're totally right. I misread and misremembered my own PR. Sheesh.